projects/effortless.md

```diff
@@ -0,0 +1,47 @@
+# Effortless
+
+The Effortless project includes the Habitat Scaffolding for Chef Infra and Chef InSpec and the things to make using them easy.
+
+**Slack Channel** #effortless
+
+**Triage Schedule**: TBD
+
+### Project Members
+
+#### Project Owners
+
+John Snow
+  - Github: [thelunaticscripter](https://github.com/thelunaticscripter)
+  - Slack: @thelunaticscripter
+
+Josh O'Brien
+  - Github: [OBrienCommaJosh](https://github.com/OBrienCommaJosh)
+  - Slack: @job
+
+Mark Dennard
```

Comment:
No offense to Mark here, but he does not have a single commit to the repo. How is he consider a project owner?

Reply:
The reason that Mark is listed as a Project Owner is this represents the Product Team from Product and Engineering. That is why the Owner list is this way with Mark as Designer, Josh as Product Manager, and myself as Engineer Lead